ecommerce/src/main/kotlin/io/hhplus/ecommerce/common/sse/SseEmitterService.kt

```
@@ -0,0 +1,119 @@
+            logger.debug("[SSE] 연결 없음: userId={}, eventType={}", userId, eventType)
+            return
+        }
+
+        try {
+            emitter.send(
+                SseEmitter.event()
+                    .name(eventType.value)
+                    .data(data)
+            )
+            logger.info("[SSE] 이벤트 전송 완료: userId={}, eventType={}", userId, eventType)
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)